lib/cache.go

```diff
@@ -0,0 +1,50 @@
+package lib
+
+import (
+	"encoding/gob"
+	"os"
+
+	migration9 "github.com/filecoin-project/specs-actors/v3/actors/migration/nv9"
+	cid "github.com/ipfs/go-cid"
+	"github.com/mitchellh/go-homedir"
+)
+
+// persist and load migration caches
+var entCachePath = "~/.ent/cache/"
```

Comment:
This will require the user to know where to manually delete this data for a reset. That's pragmatic for now, but at least print out this path in a log message in the LoadCache and PersistCache methods so that it's transparent to the user.